aemedge/blocks/channel-shopper/channel-shopper.css

```diff
@@ -26,7 +26,7 @@
     height: 20px;
 }
 
-.channel-shopper.block #channel-shopper-app button {
+.channel-shopper.block #channel-shopper-app button:not(.sc-htoDjs, .bQIqkG) {
```

Comment:
that's too fragile. Every time the code gets updated, those classes are going to change. Can you do something like "nth child" type of selection?